farcy/__init__.py

```diff
@@ -70,6 +70,7 @@ class Farcy(object):
     def __init__(self, config):
         """Initialize an instance of Farcy that monitors owner/repository."""
         # Configure logging
+        #Blah
```

Comment:
_[farcy v1.2.2](https://github.com/appfolio/farcy)_
* 9: E265 block comment should start with '# '